Changelog.md

```diff
@@ -10,6 +10,7 @@ Compiler Features:
  * Peephole Optimizer: Remove double ``iszero`` before ``jumpi``.
  * SMTChecker: Support enums without typecast.
  * SMTChecker: Support one-dimensional arrays.
+ * Optimizer: Add rule for shifts with constants (0, >=256) for Constantinople.
```

Comment:
```suggestion
 * Optimizer: Add rule for shifts by constants larger than 255 for Constantinople.
```

Reply:
I notice it says "for Constantinople". But i don't see that check in the code.. how does that work?

Reply:
I think the optimizer does not have to know - it only generates constantinople opcodes if the input is constantinople opcodes.